src/sql/parts/notebook/notebookInput.ts

```diff
@@ -131,6 +139,10 @@ export class NotebookInput extends EditorInput {
 		return this._model.providers;
 	}
 
+	public get connectionProfileId(): string {
+		return this._connectionProfileId;
```

Comment:
Please remove assignment in constructor and just use passthrough getter like other cases do
```suggestion
		return this._model.connectionProfileId;
```